aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -517,6 +539,8 @@ public class AwsProperties implements Serializable {
   public static final String LAKE_FORMATION_DB_NAME = "lakeformation.db-name";
 
   private String httpClientType;
+  private Long httpClientUrlConnectionConnectionTimeoutMs;
+  private Long httpClientUrlConnectionSocketTimeoutMS;
```

Comment:
[nit] let make this `Ms`
```suggestion
  private Long httpClientUrlConnectionSocketTimeoutMs;
```